server/src/test/proto/spine/test/event/enrichment/enrichment.proto

```diff
@@ -43,3 +44,9 @@ message ProjectCreatedEnrichmentAnotherPackageFqnAndMsgOpt {
 
     string project_name = 1 [(by) = "spine.test.event.ProjectCreated.project_id"];
 }
+
```

Comment:
Let's add a comment, which package is a target. Plus, I would mention the name of the unit test.